src/java.base/share/classes/java/nio/file/FileStore.java

```diff
@@ -96,37 +96,38 @@ protected FileStore() {
     public abstract long getTotalSpace() throws IOException;
 
     /**
-     * Returns the number of bytes available to this Java virtual machine on the
-     * file store.  If the number of bytes available is greater than
-     * {@link Long#MAX_VALUE}, then {@code Long.MAX_VALUE} will be returned.
+     * Returns the number of bytes of the file store available to this Java
+     * virtual machine as of invoking this method. If the number of bytes
+     * available is greater than {@link Long#MAX_VALUE}, then
+     * {@code Long.MAX_VALUE} will be returned.
```

Comment:
I think the original paragraph is okay. The change to the second paragraph looks good.